client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -207,13 +209,15 @@ class CommitManager(appUniqueId: String, val conf: CelebornConf, lifecycleManage
       mapId: Int,
       attemptId: Int,
       numMappers: Int,
-      partitionId: Int = -1): (Boolean, Boolean) = {
+      partitionId: Int = -1,
+      pushFailedBatches: util.Set[PushFailedBatch] = Sets.newHashSet()): (Boolean, Boolean) = {
```

Comment:
Cant this not be an immutable default ?

```suggestion
      pushFailedBatches: util.Set[PushFailedBatch] = Collections.emptySet()): (Boolean, Boolean) = {
```

Reply:
get, thanks for advice, done